include/vfn/nvme/ctrl.h

```diff
@@ -82,6 +82,8 @@ struct nvme_ctrl {
 	struct {
 		int nsqa, ncqa;
 		int mqes;
+		// Page shifts for controller, already including +12 unlike NVMe spec.
+		int mps;
```

Comment:
Since this is not really a copy of `mps` (`mqes`, `nsqa` and `ncqa` all mirror the spec interpretation), add two new members, `int pgshift` and `size_t pgsize`. We are almost always shifting it to get the page size at runtime anyway, so might as well cache the value.

Reply:
`pgshift` and `pgsize` added